tests/test_ext_marshmallow_field.py

```diff
@@ -26,7 +26,7 @@ def test_field2choices_preserving_order(openapi):
         (fields.Boolean, "boolean"),
         (fields.Bool, "boolean"),
         (fields.UUID, "string"),
-        (fields.DateTime, "string"),
+        (fields.DateTime, ["string", "integer"]),
```

Comment:
Why? This is overridden in `datetime2properties` for the cases other than ISO, which is string. Right? (You may want to add a comment here as well to explain that.)